diff --git a/CMakeLists.txt b/CMakeLists.txt
index 12320fbf..0eb430ca 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,13 +12,6 @@ include(cmake/ide.cmake)
 
 spdlog_extract_version()
 
-#---------------------------------------------------------------------------------------
-# Set default build to release
-#---------------------------------------------------------------------------------------
-if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose Release or Debug" FORCE)
-endif()
-
 project(spdlog VERSION ${SPDLOG_VERSION} LANGUAGES CXX)
 message(STATUS "Build spdlog: ${SPDLOG_VERSION}")
 
@@ -66,7 +59,7 @@ option(SPDLOG_FMT_EXTERNAL "Use external fmt library instead of bundled" OFF)
 
 if(WIN32)
 	option(SPDLOG_WCHAR_SUPPORT "Support wchar api" OFF)
-    option(SPDLOG_WCHAR_FILENAMES "Support wchar filenames" OFF)    
+    option(SPDLOG_WCHAR_FILENAMES "Support wchar filenames" OFF)
 endif()
 
 option(SPDLOG_NO_EXCEPTIONS "Compile with -fno-exceptions. Call abort() on any spdlog exceptions" OFF)
@@ -147,7 +140,7 @@ if(SPDLOG_WCHAR_SUPPORT)
  endif()
 
  if(SPDLOG_NO_EXCEPTIONS)
-	target_compile_definitions(spdlog PUBLIC SPDLOG_NO_EXCEPTIONS)	
+	target_compile_definitions(spdlog PUBLIC SPDLOG_NO_EXCEPTIONS)
 
 	target_compile_definitions(spdlog_header_only INTERFACE SPDLOG_NO_EXCEPTIONS)
 
